jobs/pacman-tenable-enricher/src/main/java/com/tmobile/cso/pacman/tenable/jobs/TenableVMVulnerabilityDataImporter.java

```diff
@@ -264,7 +264,13 @@ private UUID triggerVulnerabilityExport() throws TenableDataImportException {
 
     private long calculateTimeRange() {
         final LocalDate now = LocalDate.now();
-        LocalDate yesterday = now.minusDays(1);
+        LocalDate yesterday;
+        if(!ElasticSearchManager.indexExists("/"+TENABLE_VM_VULNERABILITY)) {
+            yesterday = now.minusDays(90);
```

Comment:
@AishwaryaKulkarni97 Can this be a job param as discussed?

Reply:
Added logic as mentioned